packages/smooth_app/lib/themes/smooth_theme.dart

```diff
@@ -16,6 +16,11 @@ class SmoothTheme {
   @visibleForTesting
   const SmoothTheme();
 
+  /// Theme color tags
+  static const String COLOR_TAG_BLUE = 'blue';
```

Comment:
Are they still used?

Reply:
Yes in the map for the old three color selection buttons

Reply:
Shouldn’t we remove them also? This PR seems to replace, isn’t?

Reply:
We currently loose the iconic status with color change. We rediscussed with Quentin on Friday. He's making visual/branding propositions for the app next week. 
I see the ability to have a 💩 brown app as somewhat of a PR liability (also I don't like brown, green and blue are both ok)

Reply:
The color picker is in dev mode, the three colors are in the normal app settings @g123k, this PR is more about adding the color picker for testing different app colors not for providing a way of customization. 

I'd say we leave the buttons in for now. Add the custom color picker for testing and when we have a fixed color (for Branding purposes) we can remove both.